core/chains/evm/config/toml/defaults/Shibarium_Mainnet.toml

```diff
@@ -0,0 +1,33 @@
+ChainID = '109'
+FinalityTagEnabled = true
+LinkContractAddress = '0x71052BAe71C25C78E37fD12E5ff1101A71d9018F'
+LogPollInterval = '1s'
+MinIncomingConfirmations = 5
+NoNewHeadsThreshold = '30s'
+RPCBlockQueryDelay = 10
+RPCDefaultBatchSize = 100
+
+[Transactions]
+ResendAfterThreshold = '1m'
+
+[BalanceMonitor]
+Enabled = true
+
+[GasEstimator]
+EIP1559DynamicFees = true
+TipCapDefault = '10 gwei'
+TipCapMin = '2 gwei'
+BumpMin = '5 gwei'
+BumpThreshold = 5
+PriceMax = '10000 gwei'
+LimitDefault = 8000000
```

Comment:
IIRC isn't this a high limit 8000000 set for CCIP? Is this (and for testnet) something that we can use the default for?